packages/suite/jest.config.js

```diff
@@ -33,7 +33,8 @@ module.exports = {
         '^@(.+)-support/(.+)': '<rootDir>/src/support/$1/$2',
         '^@(.+)-support': '<rootDir>/src/support/$1/index',
         '^@(.+)-utils/(.+)': '<rootDir>/src/utils/$1/$2',
-        '^@(.+)-utils': '<rootDir>/src/utils/$1/index',
+        // todo: could we please remove these crazy aliases?
+        '^(?!@trezor)@(.+)-utils': '<rootDir>/src/utils/$1/index',
```

Comment:
This wildcards are evil, maybe better use approach as first one for @suite-common because it this module mapper will take first match and other throw away. So if you place something like `'^@trezor/(.+)': '<rootDir>/../../suite-common/$1',` at the begining of module mapper, it should work for all @trezor packages so no-one will have issue again.

Reply:
cool, this seem to work b66f50fbd92b9dd4355cf0a3f60ef79bb40470f2 (just that I am not in suite-common)

and what about removing aliases completely? it makes more harm than good in my opionion.

Reply:
Not sure how do you want to do this now? I definitely aim to do that, but my plan is to slowly split suite into smaller packages which could replace aliases.

Another option could be replacing them with relative paths right now, but I am think this is not good solution, because it will cause lot of confusion for others and with super deep folder structure here it will be crazy anyway.